packages/angular/projects/clr-angular/src/data/datagrid/datagrid.html

```diff
@@ -20,14 +20,19 @@
                     class="datagrid-column datagrid-select datagrid-fixed-column"
                     *ngIf="selection.selectionType === SELECTION_TYPE.Multi"
                   >
-                    <span class="datagrid-column-title">
+                    <div class="clr-checkbox-wrapper">
                       <input
-                        clrCheckbox
                         type="checkbox"
+                        id="dg-select-all"
```

Comment:
So shouldn't we use a unique id here instead of a hardcorded id? This hardcoded `id` would be repeated in case there are multiple datagrids.

Reply:
Right, I forgot to change that. Good catch.